index.html

```diff
@@ -325,23 +325,34 @@
             var sublayer;
 
             // Add data layer to your map
-            cartodb.createLayer(map_object,"https://rawgit.com/openoakland/measure-g/master/viz.json")
-                .addTo(map_object)
-                .done(function(layer) {
-                    sublayer = layer.getSubLayer(1);
-                    // cartodb.vis.Vis.addInfowindow(map_object, sublayer, ['school']);
-                    // sublayer.setInteraction(true);
-                    // sublayer.infowindow.set({
-                    //     template: $('#infowindow_template').html()
-                    // });
-
-                    // sublayer.on('featureOver', function(e, latlng, pos, data) {
-                    //     alert("Hey! You clicked " + data);
-                    // });
-                })
-                .error(function(err) {
-                    console.log("error: " + err);
-                });
+            var vizUrl = "https://rawgit.com/openoakland/measure-g/master/viz.json";
+
+            $.ajax('./viz.json', {
+                dataType: 'jsonp'
```

Comment:
This is awesome! Thank you!

This line seems like the key. Not too worried about code sketchiness with this project, but wondering... do you see any way to force this data type on the server side?